docs/tutorials/earthquake_detection.ipynb

```diff
@@ -136,14 +136,29 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "nbmake": {
+     "mock": {
+      "datamodule.dataset_class.filename": "earthquakes_sample.h5",
+      "datamodule.dataset_class.url": "https://huggingface.co/datasets/DarthReca/quakeset/resolve/main/earthquakes_sample.h5"
```

Comment:
```suggestion
      "datamodule.dataset_class.url": "https://hf.co/datasets/DarthReca/quakeset/resolve/4521cf03c4bb67084c0460412785722646f2ec9d/earthquakes_sample.h5"
```
Git commit hash ensures reproducibility, in case the file somehow gets renamed or something like that.